src/FixedDataTableCell.js

```diff
@@ -11,13 +11,13 @@
  */
 
 import FixedDataTableCellDefault from 'FixedDataTableCellDefault';
-import FixedDataTableColumnReorderHandle from './FixedDataTableColumnReorderHandle';
 import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'cx';
 import joinClasses from 'joinClasses';
 import shallowEqual from 'shallowEqual';
 import { polyfill as lifecycleCompatibilityPolyfill } from 'react-lifecycles-compat';
+import ResizeReorderCell from 'ResizeReorderCell'
 
 class FixedDataTableCell extends React.Component {
```

Comment:
Should this now be a `React.PureComponent`?

Reply:
FDTCell or ResizeReorderCell ?